analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,127 @@
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from collections import OrderedDict
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+class Analyser_Merge_Milestone_FR_metropole(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        doc = dict(
+            detail = T_(
+            '''A road milestone is present but not integrated. The list of milestone comes from the CEREMA's database "RIU" in France by the Ministry of the Ecological and Inclusive Transition.'''),
+            fix = T_(
+            '''Add a node or to integrate tags if milestone already existing.'''),
+            trap = T_(
+            '''The position of the marker may be a little different than that visible on the road. Sometimes, a small white line perpendicular to the road on the emergency stop strip or the left flared strip can be seen on satellite images or mapillary's photo
+            '''))
+
+        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone not integrated'), **doc)
+        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone integration suggestion'), **doc)
+        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone update'), **doc)
+
+        self.init(
+            u"https://www.data.gouv.fr/fr/datasets/bornage-du-reseau-routier-national/",
+            u"Bornage du réseau routier national",
+            CSV(Source(u"data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2019",
+                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/fbc8b73b-a65c-486b-a710-ed22b9e4070c",
+                    separator = "\t"),
```

Comment:
u'' string no more required.

```suggestion
            "https://www.data.gouv.fr/fr/datasets/bornage-du-reseau-routier-national/",
            "Bornage du réseau routier national",
            CSV(Source("data.gouv.fr:Ministère de la Transition écologique et solidaire", millesime = "01/2019",
                    fileUrl = u"https://www.data.gouv.fr/fr/datasets/r/fbc8b73b-a65c-486b-a710-ed22b9e4070c",
                    separator = "\t"),
```